mithril-signer/tests/state_machine.rs

```diff
@@ -93,5 +96,41 @@ async fn test_create_single_signature() {
 
         .comment("signer should be able to create a single signature → Signed")
         .cycle_signed().await.unwrap()
+
+        .comment("metrics should be correctly computed")
+        .check_metrics(BTreeMap::from([
+            (
+                "mithril_signer_runtime_cycle_success_since_startup".to_string(),
+                Value::Counter(22.0),
```

Comment:
It's not easy to know where this value comes from. Could we make this more explicit?

Reply:
I based the metrics verification on the explicit number of expected number of signer and signature registrations